openapi/api_trial_system.template.yaml

```diff
@@ -0,0 +1,111 @@
+swagger: "2.0"
```

Comment:
since we're not making external reference (to `node_moduled` packages), we don't need this templating trick

Reply:
I think we do need it to "unfold" [ProblemJson and CreatedAt](https://github.com/pagopa/io-backend/pull/1119/files#diff-d272ba26db7af544f0f5dd92f9b146f922ac695169cf945c1d7ee79b65d288c4R75-R78)